images/image-set-type.html

```diff
@@ -30,6 +33,9 @@
 
     <textarea class="playable playable-css" style="height: 150px;">
```

Comment:
```suggestion
    <textarea class="playable playable-css" style="height: 155px;">
```
height: 150px;
![before](https://user-images.githubusercontent.com/74658643/134764332-792fe712-0e0a-4c3a-bff2-e7f02e0a8050.png)

height: 155px;
![after](https://user-images.githubusercontent.com/74658643/134764386-49da1a89-e707-4dfb-9306-d094af34ea2d.png)